### ISSUE TITLE
Search bar is very erratic

### ISSUE THREAD

**Issue by etrulls** [radiant-player/radiant-player-mac#554]
Click on the box from "Listen Now", no thing happens. Click on it from "My Library" and it does work. Click on it after a search result, and it does not.

Re-installed, same thing happens. Latest version running on Yosemite.

**Comment by BarakaAka1Only** [comment]
I am on Yosemite myself and i tried what you stated above and i am not getting your results it works fine for me 100% and on all skins too, have you tried on this safari itself? If you have, was it the same issue?

**Comment by ianpatrickobrien** [comment]
Have exact same issue as etrulls. Click on the search bar but nothing happens cannot enter text. Very frustrating :(
Latest version running on latest version OS X Yosemite. Have updated to latest flash and tried the html 5 option, no change.

@BarakaAka1Only - don't understand your suggestion re: running in Safari - radiant player is standalone m, that's kind of the point? 

These are the times I wish I knew about coding :/ 
Devs please look into this if you can. Thanks for a great app!

**Comment by BarakaAka1Only** [comment]
@ianpatrickobrien the reason i asked to try on safari is because Radiant Player is a web app via runs on a CustomView browser, webkit that is moreless safari

**Comment by ianpatrickobrien** [comment]
@BarakaAka1only sorry still not following, understand that google music can be accessed from a browser but thought the point of radiant is that it's a standalone app. How does one run radiant from 'within' safari?
sorry I'm not a dev, you may have figured that by now :)

**Comment by BarakaAka1Only** [comment]
@ianpatrickobrien Radiant player is a browser just like safari they are both webkit which relies on the Webkit.framework, Radiant Player has built a Webview using a CustomView to handle Protocols that Radiant Player uses; take for example how the custom styles work in the preferences same with the cookies and updates for Radiant Player. (that's the short end) 

To be frank Radiant Player uses embedded Safari.

Also if you were to right-click in Radiant Player and click Inspect Element and typed below `window.location.href = 'http://www.google.com`
and hit enter it will take you to Google 

![image](https://cloud.githubusercontent.com/assets/2947494/15203684/ab5b1852-17cb-11e6-8a61-85c045936308.png)

Hopefully that gives you an idea and take a look here https://github.com/radiant-player/radiant-player-mac/issues/549#issuecomment-213447663

**Comment by jacobwgillespie** [comment]
Yep, Radiant essentially just wraps a Safari view inside a desktop app.  Here's a few other questions for both of you experiencing this issue:

What are your settings from Preferences > Appearance for "Use custom title bar"?

What version of OS X are you running?

**Comment by ianpatrickobrien** [comment]
@BarakaAka1only thank you for your explanation. Everything working fine in Safari.
My issue with Radiant player is the same as @etrulls - search bar works once after going to 'my library', after that it stops working, click on the title bar and nothing happens, impossible to enter text :(

**Comment by ianpatrickobrien** [comment]
@jacobwgillespie have tried custom title bar on/off and every combination of skins, nothing solves the issue.
Running latest OS X Yosemite 10.10.5

**Comment by ianpatrickobrien** [comment]
...and latest flash

**Comment by etrulls** [comment]
@BarakaAka1Only Working fine on safari.

@jacobwgillespie I deleted the preference files while reinstalling, so this is with the defaults. I've tried the appearance options (no title bar, custom styles), same behaviour.

**Comment by danielhq** [comment]
Search doesn't work for me either.
Im on OSX Yosemite 10.10.4
opening the console it gives me this error:
TypeError: null is not an object (evaluating 'document.querySelector('#material-one-middle input.sj-search-box').select')

Might be related.

**Comment by padrediego** [comment]
Same issue here, Yosemite 10.10.5

**Comment by chrismou** [comment]
We did some fixes CSS fixes for the search box in El Capitan a while ago. Might be as simple as tweaking them to also apply to Yosemite: https://github.com/radiant-player/radiant-player-mac/pull/537/files#diff-acbad8c7fdbc679522c04fe703e40c26R939 .  Will need to knock out a dev build though and get someone with Yosemite to test it, as I don't have a copy to hand.

As a side note, Google only officially supports the latest and previous version of Safari (which is what Radiant currently uses behind the scenes to render the site): https://support.google.com/a/answer/33864?hl=en -- I'm not sure what the highest version is that's available on Yosemite, but there'll inevitably come a time when it just won't work anymore.  I'm sure you guys have your reasons for sticking to Yosemite, but worth bearing in mind.

As a side, side note, we're also working on a cross-platform version of Radiant, built on Electron (https://github.com/radiant-player/radiant-player-electron) which will solve this problem, as it uses Chrome behind the scenes. It's not quite ready for prime time though, so bear that in mind if you decide to give it a go and come across any issues...!

**Comment by BarakaAka1Only** [comment]
Not sure cause I am on Yosemite myself and yeah for my own reasons, and the search works perfectly for me i haven't had any issues so i am not sure what the case is with this matter. 

I can look more into it, Google does changes from here and there just like the css issues with Flex

as of now with minor testing if i manually place in

`input#input.style-scope.sj-search-box {
   position: relative;
   z-index: -1;
}`

then it emulates the issue but if anyone here with this issue doesn't mind, 

pasting in this mini js code to apply a test style for the search box to see if it fixes it or not

in the console when inspecting an element

`function createStyle(e){var t=document.createDocumentFragment(),n=document.createElement("div");for(n.innerHTML=e;n.firstChild;)t.appendChild(n.firstChild);return t}var radiant=createStyle('<style id="search-boxtest">input#input.style-scope.sj-search-box{position: relative;z-index: 16 !important;}</style>');document.body.insertBefore(radiant,document.body.childNodes[0]);`

Hopefully that's the issue with the z-index

as too @danielhq with that js error has anyone else encountered that?
